test/functional/apps/discover/_errors.ts

```diff
@@ -38,8 +39,8 @@ export default function ({ getService, getPageObjects }) {
 
     describe('invalid scripted field error', () => {
       it('is rendered', async () => {
-        const isFetchErrorVisible = await testSubjects.exists('discoverFetchError');
-        expect(isFetchErrorVisible).to.be(true);
+        const toast = await toasts.getToastElement(1);
+        expect(toast).not.to.be(undefined);
```

Comment:
I wonder if it would also be possible to test what error was displayed? would be helpful in cased like recently when the error reply returned changed, and a Painless error became a ordinary one.

Reply:
Added validation specific to the painless toast